src/workers/search/project-search.js

```diff
@@ -4,32 +4,65 @@
 
 // Maybe reuse file search's functions?
 
+import getMatches from "./get-matches";
+
 export function findSourceMatches(source, queryText) {
-  const { id, loadedState, text } = source;
+  const { loadedState, text } = source;
   if (loadedState != "loaded" || !text || queryText == "") {
     return [];
   }
 
+  const modifiers = {
```

Comment:
I guess those should be picked up from the state instead of being hard coded here

Reply:
For now, we don't have them for project search